doc/releases/nautilus.rst

```diff
@@ -0,0 +1,533 @@
+v14.1.0 Nautilus (release candidate 1)
+======================================
+
+.. note: We expect to make a msgr2 protocol revision after this first
+   release candidate.  If you upgrade to v14.1.0 *and* enable msgr2,
+   you will need to restart all daemons after upgrading to v14.1.1 or
+   any other later nautilus release.
+
+.. note: These are draft notes for the first Nautilus release.
+
+Major Changes from Mimic
+------------------------
+
+- *Dashboard*:
```

Comment:
The Ceph Dashboard has gained a lot of new functionality:

* Support for multiple users / roles
* SSO (SAMLv2) for user authentication
* Auditing support
* New landing page, showing more metrics and health info
* I18N support
* Swagger REST API

New Ceph management features include:

* OSD management (mark as down/out, change OSD settings, recovery profiles)
* Cluster config settings editor
* Ceph Pool management (create/modify/delete)
* ECP management
* RBD mirroring configuration
* Embedded Grafana Dashboards (derived from Ceph Metrics)
* CRUSH map viewer
* NFS Ganesha management
* iSCSI target management (via ceph-iscsi)
* RBD QoS configuration
* Prometheus Alert Management

Reply:
@LenzGr  maybe change "Swagger REST API" to "REST API documentation with Swagger UI"?

Reply:
Agreed, that sounds better. @liewegas - could you please update this accordingly?